seed-job/template.yaml

```diff
@@ -55,76 +55,89 @@ objects:
+                                  sh "cat rpm-verify"
+                                },
+                                "Miscellaneous updates": {
+                                  sh (returnStdout: true, script: "docker run --rm --volumes-from `docker ps -qf name=${daemonset_name}` --entrypoint /bin/python ${image_name} /opt/scanning/misc_package_updates.py all > misc-updates 2>&1", returnStatus: true)
+                                  sh "cat misc-updates"
+                                },
+                                "Container capabilities": {
+                                  def run_label = sh (returnStdout: true, script: "docker inspect ${image_name} --format '{{ index .Config.Labels \"RUN\" }}'")
+                                  sh (returnStdout: true, script: "docker run --rm --volumes-from `docker ps -qf name=${daemonset_name}` --entrypoint /bin/python ${image_name} /opt/scanning/container-capabilities.py ' ${run_label} ' > capabilities 2>&1", returnStatus: true)
+                                  sh "cat capabilities"
+                                }
+                            )
+                        }
+                        stage("Push image to registry") {
+                            sh "docker tag ${image_name} ${image_name_with_registry}"
+                            sh "docker push ${image_name_with_registry}"
+                        }
+                        // this stage should be triggered after image is delivered to the registry
+                        stage('Remove the image'){
+                            sh "docker rmi ${image_name_with_registry}"
+                            sh "docker rmi ${image_name}"
+                            // image lint - build - scan - deliver is successful
+                            success = true
+                        }
                     }
-                    // this stage should be triggered after image is delivered to the registry
-                    stage('Remove the image'){
-                        sh "docker rmi ${image_name_with_registry}"
-                        sh "docker rmi ${image_name}"
+                    finally {
+                        if (success==true) {
+                            sh "echo -e 'Build request for ${image_name_with_registry} was successful' | mail -r container-build-reports@centos.org -S smtp=smtp://mail.centos.org -s 'SUCCESS: Container build ${image_name_with_registry} is complete' ${NOTIFY_EMAIL} "
+                        } else {
+                            sh "echo -e 'Build request for ${image_name_with_registry} was unsuccessful' | mail -r container-build-reports@centos.org -S smtp=smtp://mail.centos.org -s 'FAILED: Container build ${image_name_with_registry} has failed' ${NOTIFY_EMAIL} "
```

Comment:
we took the smtp in parameter, then why we are hard coding it here? both the from id and smtp is hard coded

Reply:
Thanks for the catch. Fixed.